frontend/src/home/Home.tsx

```diff
@@ -452,6 +461,37 @@ class HomeComponent extends React.Component<Props, HomeState> {
     }
   }
 
+  renderTransfer() {
+    // If a user is currently logged in, wait until plans are fetched to render
+    if (this.props.token && this.props.userId) {
```

Comment:
Looks like this component takes in `isLoggedIn` from state, so this can be replaced with `this.props.isLoggedIn` just to make things a little more readable 😃